web-client/src/Api.elm

```diff
@@ -273,7 +349,7 @@ mutationAddHabit createHabit =
         }"""
                 |> Util.templater templateDict
     in
-    graphQLRequest queryString <| Decode.at [ "data", "add_habit" ] Habit.decodeHabit
```

Comment:
Oh god no! I **hate** when `let` indents, it makes the code nested for no reason. If you use the alpha version of elm-format then it won't indent `let`s. Also in general you don't want to be changing the indentation in the same commit as the code, cause it adds tons of unrelated diffs